assets/src/blocks/ENForm/ENForm.js

```diff
@@ -78,11 +78,12 @@ export class ENForm extends Component {
                 { __( 'Check your EngagingNetworks settings!', 'planet4-engagingnetworks-backend' ) }
               </InlineFormFeedback>
           }
-
+          <br />
           <SelectControl
             label={__( '- Select Goal -', 'planet4-engagingnetworks-backend' )}
             value={this.props.enform_goal}
             options={[
+              { label: '--- Select Goal ---', value: 'not set' },
```

Comment:
maybe we could make the label translatable ? So that it can correspond to the label